### PR TITLE
CORTX-30825: Codacy Fix [merge feature branch to main]

### PR DESCRIPTION
**Pull request by rohitkolapkar**
Signed-off-by: Rohit Kolapkar <rohit.j.kolapkar@seagate.com>

# Problem Statement
- [Fix codacy issues and follow coding standards](https://jts.seagate.com/browse/CORTX-30825)

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

# Dev Testing
1. CSM User Management

image

2. All Codacy issues fixed on feature branch 
image

-----
[View rendered .github/pull_request_template.md](https://github.com/Seagate/cortx-manager/blob/codacy_fix/.github/pull_request_template.md)
[View rendered CODE_OF_CONDUCT.md](https://github.com/Seagate/cortx-manager/blob/codacy_fix/CODE_OF_CONDUCT.md)
[View rendered CORTXManagerQuickstartGuide.md](https://github.com/Seagate/cortx-manager/blob/codacy_fix/CORTXManagerQuickstartGuide.md)
[View rendered CodingStyle.md](https://github.com/Seagate/cortx-manager/blob/codacy_fix/CodingStyle.md)
[View rendered README.md](https://github.com/Seagate/cortx-manager/blob/codacy_fix/README.md)
[View rendered SECURITY.md](https://github.com/Seagate/cortx-manager/blob/codacy_fix/SECURITY.md)

### CODE REVIEW

**Comment by gauravchaudhari02** [comment]
retest this please

File: CodingStyle.md [hunk 1]
```diff
@@ -48,18 +51,20 @@ We've listed the coding style that is specific to any submissions made to the CO
   import time
   ```
 
-- Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
-- Imports should be grouped in the following order:
-  - Standard library imports.
-  - Related third party imports.
-  - Local application/library specific imports.
-- You should put a blank line between each group of imports.
+-   Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
```
[the remark this answers is not in the document]
Comment (rohitkolapkar) [reply CodingStyle.md]: resolved.

**Comment by rohitkolapkar** [comment]
3n deployment success: https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/4275/

custom ci success: https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/custom-ci/6081/

**Review by avrnv — APPROVED** [review]

**Review by pawankumarsrivastava — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -33,9 +33,9 @@ We take community very seriously and we are committed to creating a community bu
 
 ## Reach Out To Us
 
-- Join our CORTX-Open Source Slack Channel to interact with your fellow community members and gets your questions answered. [![Slack Channel](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?)
-- If you'd like to contact us directly, drop us a mail at cortx-questions@seagate.com.
+-   Join our CORTX-Open Source Slack Channel to interact with your fellow community members and gets your questions answered. [![Slack Channel](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?)
```
[the remark this answers is not in the document]
Comment (rohitkolapkar) [reply README.md]: resolved

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -14,22 +14,25 @@ appearance, race, religion, or sexual identity and orientation.
...
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
- advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
- address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
- professional setting
+*   The use of sexualized language or imagery and unwelcome sexual attention or
```
[the remark this answers is not in the document]
Comment (kupranay) [reply CODE_OF_CONDUCT.md:25]: As per discussion ignore this instance. RE team will stop scanning this specific file

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -14,22 +14,25 @@ appearance, race, religion, or sexual identity and orientation.
...
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
- address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
- professional setting
+*   The use of sexualized language or imagery and unwelcome sexual attention or 
+advances
+
+*   Trolling, insulting/derogatory comments, and personal or political attacks
```
[the remark this answers is not in the document]
Comment (kupranay) [reply CODE_OF_CONDUCT.md:28]: As per discussion ignore this instance. RE team will stop scanning this specific file

**Pull request merged**